scanner/src/main/java/edu/ucr/cs/riple/scanner/Scanner.java

```diff
@@ -54,6 +54,7 @@
     summary = "Serialized type based call/field graph.",
     tags = BugPattern.StandardTags.STYLE,
     severity = SUGGESTION)
+@SuppressWarnings("BugPatternNaming")
```

Comment:
Is the warning here that we don't need `name = [...]` if its the same name as the class? If so, why suppress instead of removing `name`? Or is the error something else?

Reply:
Yes that error is due to the issue you mentioned, but there is no constructor which does not require the name. When I did that, there was an error that name is missing.

Reply:
Weird, I wonder if it's a discrepancy between `@BugPattern` requirements across different EP versions... is it the same version of error prone that failes with and without the name locally or are they different CI tasks?

Reply:
We have this suppression in NullAway as well:

https://github.com/uber/NullAway/blob/master/nullaway/src/main/java/com/uber/nullaway/NullAway.java#L150

There it was because we want to be able to build against Error Prone versions older than 2.11, and there was some issue with setting the bug pattern name consistently in older versions (I can't remember all the details).